services/core-api/app/api/mines/reports/models/mine_report_req_permit_condition_xref.py

```diff
@@ -5,7 +5,7 @@
 from app.api.utils.models_mixins import Base, AuditMixin, SoftDeleteMixin, HistoryMixin
 from app.extensions import db
 
-class MineReportReqPermitConditionXref(Base, AuditMixin, SoftDeleteMixin, HistoryMixin):
+class MineReportReqPermitConditionXref(SoftDeleteMixin, AuditMixin, HistoryMixin, Base):
```

Comment:
this is where I learned something new about python inheritance. Having Base before SoftDeleteMixin made it so it called the Base delete function. 🤦‍♀️

Reply:
Interesting, I didn't realize the before either